admin_manual/configuration_server/bruteforce_configuration.rst

```diff
@@ -22,6 +22,16 @@ The maximum delay is 25 seconds.
 After a successful login the attempts will be cleared. And once a user is
 properly authenticated they will no longer be hit by the delay.
 
+Exclude IP addresses from brute force protection
+------------------------------------------------
+
+It's possible to exlude IP addresses from the brute force protection.
+
+- Enable the bruteforcesettings app
+- Login as admin and go to Administration settings -> Security
+
+Note that any excluded IP address can perform authentication attempts without any throttling.
+Its best to exclude as few IP addresses as you can, or even none at all. 
```

Comment:
Are issues with proxies mentioned on this page and that instead of excluding a (reverse) proxy it should be configured as trusted proxy?

Reply:
Yep, it's mentioned in the troubleshooting section below: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/bruteforce_configuration.html#troubleshooting

Shall we flip troubleshooting and exclude?

Reply:
yeah I would say so

Reply:
don't know the exact syntax atm, but maybe we make this a warning box as well

```suggestion
.. warn::
    Its best to exclude as few IP addresses as you can, or even none at all. 
```